tests/test_opnieuw.py

```diff
@@ -110,6 +115,32 @@ def test_mixed_states(self) -> None:
         with retry_immediately("bar_retry"):
             self.test_retry_with_waits()
 
+    @retry(
+        retry_on_exceptions=ValueError,
+        max_calls_total=4,
+        retry_window_after_first_call_in_seconds=10,
+    )
+    def namespaced_retry_fixed_backoff(self) -> None:
+        self.counter += 1
+        if self.counter == 4:
+            raise CustomBackoffThrottleError(seconds=1)
+        raise ValueError
+
+    def test_fixed_backoff_reset_max_calls(self):
+        self.counter = 0
+        start = time.monotonic()
+
+        with retry_immediately():
```

Comment:
Hmm, so this does mean that backoff throttle waits are not included in the `retry_immediately`. I would expect them to be skipped as well. This test would still be valuable because it tests that we can make twice the number of calls, due to the counter reset.

Reply:
Yeah, that is a little unexpected :+1: